packages/firmware_notifier/analysis_options.yaml

```diff
@@ -0,0 +1,45 @@
+include: package:lints/recommended.yaml
+
+analyzer:
+  exclude:
+    - "**/*.freezed.dart"
+    - "**/*.g.dart"
+    - "**/*.mocks.dart"
+    - "**/l10n/*.dart"
+
+  strong-mode:
+    implicit-casts: false
+
+linter:
+  rules:
+    always_declare_return_types: true
```

Comment:
does this have different rules or should we let it use the top-level analysis options to get away with one file?

Reply:
one file is enough - those were just leftovers from the standalone package template before I moved it into this repo :)